build/curl/build.sh

```diff
@@ -32,6 +32,8 @@ CONFIGURE_OPTS="
     --with-ca-bundle=/etc/ssl/cacert.pem
     --with-ca-path=/etc/ssl/certs
     --with-openssl
+    --disable-manual
+    --without-libpsl
```

Comment:
Is this something that we should be looking at in the future due to the implications of using the PSL? I assume it was intentional that we weren't using it in the past.

Reply:
Curl in OmniOs/Helios has never been built with libpsl, but in 8.6.0 they made it a fatal error if it isn't found. From the release notes:

> To force users to explictily disable it if they really don't want it used and make it harder to accidentally miss it.

The decision was also blogged about at https://daniel.haxx.se/blog/2024/01/10/psl-in-curl/

Unfortunately, libpsl does not build on illumos, or on pretty much any non-linux system due to non-portable dependencies, and nobody has done the work to bring it into omnios/helios.

--disable-manual stops all of the man pages being compiled into the binary.

Reply:
Apparently libpsl has improved somewhat since the last time I looked at it, and it has now been added to omnios bloody where it is being used by wget, wget2 and curl.

I'll look at backporting that into helios separately to this PR.

Reply:
I had found that same set of release notes, but had just wanted to confirm and make sure it was intentional given the blog entry. Thanks for clarifying.